### ISSUE TITLE
Add an `.nvmrc` file

### ISSUE THREAD

**Issue by rogierslag** [diegomura/react-pdf#158]
Currently it is a bit unclear on which node version you develop. I just tried using 8.5 but that messes up all the indenting in snapshots.

Adding an `.nvmrc` file would allow everybody to run `nvm use` to select the correct version

**Comment by jbovenschen** [comment]
Don't know if this is a solution, this means every one who wants to contribute to this project need to have NVM setup on his machine. Would it help if we add a engines: [], with supported versions in the package.json in the root of the project?

**Comment by nkoder** [comment]
> every one who wants to contribute to this project need to have NVM setup on his machine

I'm quite sure it's not true :-) When `.nvmrc` is provided:
* you can use it with nvm to setup proper node version,
* you can still set another node version with nvm,
* you can have no nvm on your machine and install/use node in the version suggested in `.nvmrc` (after just looking into it to know the recommendation),
* you can have no nvm on your machine and install/use node in any other version.

In other words: I find having `.nvmrc` beneficial as it forces noone to use nvm and helps many who want to use it. And it's kind of documentation saying "this node version is preferred one".

In my projects I have both `.nvmrc` and a section in README saying:
> Use [nvm](https://github.com/creationix/nvm)
   to activate Node in version specified in `.nvmrc`:
    ```bash
    nvm install
    ```
    Alternatively you can just install proper Node version manually.
